deploy/cluster-monitoring-config/openshift-user-workload-monitoring.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-workload-monitoring-config
+  namespace: openshift-user-workload-monitoring
```

Comment:
Why are we configuring this instead of providing RBAC for dedicated-admins to set this up?  Not all customers will want this running on their cluster.  I think the thing we need to do is simply set enableUserWorkload: true

Reply:
`enableUserWorkload: true` will deploy this either way. This configmap definition is just to ensure all the components run on worker nodes only. If we don't want to automatically deploy UWM for all 4.6+ clusters, we'll need to fully close the PR and gate it behind an ocm flag.

Reply:
Talked on slack.  Reason for this CM was to put the prom operator pod on worker (by default is deployed to masters).  But also brought up that if we manage the CM via SSS it cannot be modified by customers.  So it needs to be pulled out.